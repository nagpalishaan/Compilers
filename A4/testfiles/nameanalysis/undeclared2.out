Error at line 3: expected type does not match. Type: lang.ast.BoolType@c86b9e3 != Expected type: lang.ast.IntType@757277dc
Error at line 3: symbol 'b' is not declared
