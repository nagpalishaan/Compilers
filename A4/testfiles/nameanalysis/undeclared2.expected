Error at line 3: symbol 'b' is not declared
